fuse_doc/doc/getting_started.rst

```diff
@@ -0,0 +1,253 @@
+
+.. code-block:: xml
+
+  <launch>
+    <param name="/use_sim_time" value="true"/>
+
+    <node name="state_estimator" pkg="fuse_optimizers" type="fixed_lag_smoother_node">
+      <rosparam command="load" file="$(env HOME)/fuse_tutorials/fuse_simple_tutorial.yaml"/>
+    </node>
+  </launch>
+
+Basic Configuration
+*******************
+
+Our initial configuration will simply fuse a single wheel odometry sensor. Note that this example will not produce any output. We'll cover that later in the tutorial. Open up `fuse_simple_tutorial.yaml` and paste this into it:
+
+.. code-block:: yaml
+
+  # Fixed-lag smoother configuration
+  optimization_frequency: 20
+  transaction_timeout: 0.01
+  lag_duration: 0.5
+
+  solver_options:
+    max_num_iterations: 5
+
+  motion_models:
+    unicycle_motion_model:
+      type: fuse_models::Unicycle2D
+
+  unicycle_motion_model:
+    #                         x      y      yaw    vx     vy     vyaw   ax   ay
+    process_noise_diagonal: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.1, 0.1]
+    scale_process_noise: true
+
+  sensor_models:
+    initial_localization_sensor:
+      type: fuse_models::Unicycle2DIgnition
+      motion_models: [unicycle_motion_model]
+      ignition: true
+    odometry_sensor:
+      type: fuse_models::Odometry2D
+      motion_models: [unicycle_motion_model]
+
+  initial_localization_sensor:
+    publish_on_startup: true
+    #                x      y      yaw    vx     vy     vyaw    ax     ay
+    initial_state: [0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000, 0.000]
+    initial_sigma: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.100]
+
+  odometry_sensor:
+    topic: "odom"
+    twist_target_frame: "base_footprint"
+    linear_velocity_dimensions: ['x', 'y']
+    angular_velocity_dimensions: ['yaw']
+
+  publishers:
+    filtered_publisher:
+      type: fuse_models::Odometry2DPublisher
+
+  filtered_publisher:
+    topic: "odom_filtered"
+    base_link_frame_id: "base_footprint"
+    odom_frame_id: "odom"
+    map_frame_id: "map"
+    world_frame_id: "odom"
+    publish_tf: true
+    publish_frequency: 10
+
+There's a lot to unpack here, so we'll look at one section at a time.
+
+.. code-block:: yaml
+
+  # Fixed-lag smoother configuration
+  optimization_frequency: 20
+  transaction_timeout: 0.01
+  lag_duration: 0.5
+
+  solver_options:
+    max_num_iterations: 5
+
+In this section, we specify the `optimization_frequency`, which is the how often we run our solver and produce a state estimate.
+
+We also specify the `transaction_timeout`, which specifies how long we wait for motion models to be generated when adding constraints to the graph. If this time is exceeded, the constraints are not added to the graph.
+
+The `lag_duration` parameter specifies how much our state estimate lags behind the current time. This ensures that all measurements have been received before we produce a state estimate for a given time.
```

Comment:
> This ensures that all measurements have been received *before* we produce a state estimate

This is more the size of the smoothing window. Variables added to the fixed-lag smoother will stay in the graph at least `lag_duration` seconds. After that time, old variables are removed/marginalized out. Updated estimates for all variables within the lag window are generated at `optimization_frequency` hz. The current text makes me think that state estimates are delayed by `lag_duration` seconds, which is not the case.

Reply:
OK, I'll fix that.